tests/integration_tests/security/test_vulnerabilities.py

```diff
@@ -59,6 +59,7 @@ def configure_microvm(
     else:
         microvm = factory.build(kernel, rootfs)
 
+    microvm.help.enable_console()
```

Comment:
is this needed?

Reply:
Nope, but it is extremely useful for debugging.

Reply:
Sorry, what I meant: do we still need it? I assume it is left from when you were debugging, but not it can be removed.